test/cmdlineTests/standard_binaries_requested/output.json

```diff
@@ -1 +0,0 @@
-{"contracts":{"A":{"C":{"evm":{"gasEstimates":{"creation":{"codeDepositCost":"19800","executionCost":"75","totalCost":"19875"},"external":{"f()":"122"}}}}}},"sources":{"A":{"id":0}}}
```

Comment:
All requests that trigger binary generation will either output the bytecode that will change volatilely for prerelease versions or the gas estimates, which will also change volatilely for prerelease versions, so I don't see a good way to keep this test.